tests/e2e/integrations/test_huggingface.py

```diff
@@ -281,7 +281,7 @@ def test_non_monitoring_runs_creation(self, environment, project, common_tag):
         # then
         runs = project.fetch_runs_table(tag=common_tag).to_rows()
         assert len(runs) == 1
-        assert runs[0].get_attribute_value("monitoring/cpu") is not None
```

Comment:
Can't we count the number of namespaces under `monitoring/cpu` instead? It should be easy with `get_structure()`

Reply:
@Raalsky Check out my solution - not exactly what you propose, but maybe still ok? Was not sure what you meant as there is no `monitoring/cpu` namespace, right? There is only `monitoring/{some_hash_value}/cpu`